core/common/src/thrift/file_system_master.thrift

```diff
@@ -154,6 +154,9 @@ service FileSystemMasterClientService extends common.AlluxioService {
 
```

Comment:
since you are writing javadoc for the specific methods, do you also want to document exceptions that can happen in general (e.g. permission denied, etc.)?